modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -58,10 +58,37 @@ Card Border:: Specifies the card border thickness.
 Card Border Radius:: Specifies the card border radius. A larger radius makes the cards look more rounded.
 Card Shadow:: Specifies the shadow beneath each card.
 Message If Empty:: Shows a message if the carousel has no APIs. Default is `There are no APIs available right now.`
-Filter Condition (Advanced):: Limits the APIs shown in the carousel to those matching a standard Salesforce Object Query Language (SOQL) search. SOQL searches use package names, field names such as "API name" or "API description," and comparison of strings to the values of those fields.
+Filter Condition (Advanced):: Causes the carousel to show only APIs that match the given search.
```

Comment:
please indicate that 'the syntax for this field is described in the "Filter conditions" section below'

Reply:
Fixed.